ua_parser/user_agent_parser.py

```diff
@@ -196,6 +196,10 @@ def Parse(self, user_agent_string):
         return device, brand, model
 
 
+MAX_CACHE_SIZE = 20
```

Comment:
This is the same size as what `urlparse` uses. I don't have a strong opinion about more or less. This can be changed, obviously, at runtime with `user_agent_parser.MAX_CACHE_SIZE = 100` or something.

```python
>>> import urlparse
>>> urlparse.MAX_CACHE_SIZE
20
```